src/components/ImageWithErrors/ImageWithErrors.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useIntl, defineMessages } from 'react-intl';
+import { Image } from '@plone/volto/components';
+const messages = defineMessages({
+  image_error: {
+    id: 'Error - Image not found',
+    defaultMessage: 'Errore: immagine non trovata',
+  },
+});
+const ImageWithErrors = (props) => {
+  const intl = useIntl();
+  return (
+    <Image
+      {...props}
+      onError={({ currentTarget }) => {
+        currentTarget.onerror = null; // prevents looping
+        // currentTarget.src="image_path_here";
+        currentTarget.parentElement.innerHTML = intl.formatMessage(
```

Comment:
Questo potrebbe sostituire l'intero contenuto del padre, o sbaglio?
Penso a qualcosa del tipo:

```html
<div>
  <img src="rotto" />
  <p>Testo</p>
</div>
```

Reply:
quello sostituisce completamente il tag img, con il messaggio, ovvero l'html diventa  qualcosa del tipo

`<div>Errore: immagine non trovata</div>`

Reply:
Esatto, intendo dire che qualsiasi altra cosa che stia all'interno del parentElement assieme all'immagine verrà eliminata. Ho creato un esempio base qui, vedi che il `p` sparisce. https://jsfiddle.net/u8k417zt/8/

Reply:
Cosa ne pensi di questo approccio? https://jsfiddle.net/u8k417zt/9/

Reply:
ah.. capito! 
si meglio, poi col css si può mettere l'immagine 'display none' e aggiungere una stringa con uno pseudo element. 

Oppure, avevo visto anche esempi dove facevano il replace dell'src con una immagine statica con il simbolo dell'immagine rotta, se ci può piacere

Reply:
Sì quella dell'immagine rotta di default è una cosa che stavamo ragionando anche con @deodorhunter, che abbiamo parlato brevemente a voce di questa PR e ci sembra un'alternativa molto valida. Un piccolo svg o una piccola immagine in base64 sarebbe più che sufficiente.
Alla fine usando questo metodo si può cambiare qualsiasi attributo, anche cambiare tipo l'src o qualsiasi altra cosa.
Martina mi mostrava anche una ulteriore opzione che ti riporto perché magari è interessante:

```
[error, setError] = useState(false)
const errorHandler = () = > setError(true)
...
<img src={error ? fallback : src} onerror={errorHandler} {...rest} />
```

Reply:
si esatto.. 
dato che onerror è tutto js, mi sembra superfluo sfruttare le variabili dello state. 
provo mettendo una immagine in base64 cosi semplifichiamo anche le cose